src/Material/Textfield.elm

```diff
@@ -282,25 +280,29 @@ defaultModel =
 
 {-| Component actions. `Input` carries the new value of the field.
 -}
-type Msg
+type Msg m
   = Blur
   | Focus
   | Input String
+  | Multi (Dispatch.Msg m)
 
 
 {-| Component update.
 -}
-update : Msg -> Model -> Model
+update : Msg msg -> Model -> (Model, Cmd msg)
 update action model =
   case action of
-    Input str -> 
-      { model | value = str }
-      
+    Multi msg ->
+      (model, Dispatch.forward msg)
```

Comment:
This should be 

    (Nothing, Dispatch.forward msg)

Unfortunately, Parts doesn't support that yet. I've opened an issue and will fix it shortly; see debois/elm-parts#10.